cinder/backup/chunkeddriver.py

```diff
@@ -875,6 +886,11 @@ def delete_backup(self, backup):
         LOG.debug('delete %s finished.', backup['id'])
 
 
+def use_sap_restore(volume_file):
+    class_name = volume_file.__class__.__name__
+    return class_name and "VmdkReadHandle" in class_name
```

Comment:
When don't we have a class name or in other words why do we have to check it here? Do we have code here to make the tests happy instead of letting the tests provide proper, production-like values?